### DISCUSSION TITLE
paint result area all appears to be in black or white

### DISCUSSION THREAD

**Discussion by heing8848** [Sanster/IOPaint#497] [Q&A]
After painting the area which need to be inpaint, no matter what model i use (such as runwayml/stable-diffusion-inpainting  ,   stabilityai/stable-diffusion-2-inpainting    ,     Fantasy-Studio/Paint-by-Example     ,      saik0s/realistic_vision_inpainting ), the painted area all result in black or sometime cover in white color. (on cpu and Cuda result
![Screenshot 2024-03-27 095156](https://github.com/Sanster/IOPaint/assets/53251744/54382eee-45dc-40c9-95c5-1d2273b6ef83)
![Screenshot 2024-03-27 095500](https://github.com/Sanster/IOPaint/assets/53251744/6695c556-6645-4625-9247-9103afb07ea4)
![Screenshot 2024-03-27 100019](https://github.com/Sanster/IOPaint/assets/53251744/b996bed7-82a4-4328-9c9d-3a446ecae43b)
![Screenshot 2024-03-27 100118](https://github.com/Sanster/IOPaint/assets/53251744/ef232bb1-c624-406c-974d-f93204f12900)
![Screenshot 2024-03-27 100151](https://github.com/Sanster/IOPaint/assets/53251744/57d14229-0da9-4cfb-ab97-53d4caaf70b7)
 same)  Not sure what cause this. on the other hand, only erase inpaint model functions properly. Need some help please!!!

**Comment by Sanster** [comment]
<img width="676" alt="image" src="https://github.com/Sanster/IOPaint/assets/3998421/b4f86d6c-bdb3-46f0-9f9b-12b5bb0072ab">

According to your log, it is because of an issue with the nsfw checker. You can add this parameter to start the service. `--disable-nsfw-checker`

**Reply by heing8848** [reply]
thanks for the suggestion, but it seem that its still not working for me, can you please a look. not sure if there is othr setting that might affect the result

**Reply by heing8848** [reply]
hi

**Reply by Sanster** [reply]
What graphics card are you using? You can try `--no-half`, some graphics cards may need this parameter.

**Reply by heing8848** [reply]
i am using a gtx 1060 laptop

**Reply by heing8848** [reply]
but i use cpu it also appears in white or black

**Reply by heing8848** [reply]
thanks a lot, it actually works

**Comment by heing8848** [comment]
![image](https://github.com/Sanster/IOPaint/assets/53251744/48b41054-f8f3-48e8-aad2-7d2b23231abd)
![image](https://github.com/Sanster/IOPaint/assets/53251744/ac347f80-6c44-4f02-94d7-17505e1e1663)
![image](https://github.com/Sanster/IOPaint/assets/53251744/2db24005-0f3c-4e9e-affe-eb24914b18c4)
![image](https://github.com/Sanster/IOPaint/assets/53251744/565f4368-3950-4e53-8385-ba276b592e5d)
![image](https://github.com/Sanster/IOPaint/assets/53251744/1e7e3b8b-d3b9-4a06-9edd-1517664aaca2)
it still appears in black
